README.md

```diff
@@ -1,17 +1,52 @@
 # White-labeling the docs
 
-This project is a documentation for the elastic.io platform. In order to
-provide a white-labeled documentation to your customers, please follow
-the following steps:
+This project is a documentation for the elastic.io platform. In order to provide a white-labeled documentation to your customers, please follow the following steps: 
+
 
 1. Fork this repository
 2. Customize [variables](_data/tenant.yml) used in documentation pages
 3. Customize the [styles](./assets/css/common.css)
-4. Customize your [Algolia](https://www.algolia.com/) configuration in the
-[_config.yml](./_config.yml) file (`application_id`, `index_name` and `search_only_api_key`) or delete them if you don't need search
-5. DO NOT edit any content to avoid merging conflicts
-6. Pull changes from the original repository regularly so that your docs
-are up-to-date
+4. Customize your [Algolia](https://www.algolia.com/) configuration in the [_config.yml](./_config.yml) file (application_id, index_name and search_only_api_key) or delete them if you don't need search
+5. Customize CNAME file.
```

Comment:
This is only required if you host it on GitHub but it is completely fine to host the docs on your servers so that this point is optional. We should mention that

Reply:
just put this in